annotations/pdf.go

```diff
@@ -1,43 +1,41 @@
 package annotations
 
 import (
-	"fmt"
+	"bytes"
 	"os"
 
-	"github.com/jung-kurt/gofpdf"
 	"github.com/juruen/rmapi/archive"
-	"github.com/juruen/rmapi/encoding/rm"
+	"github.com/unidoc/unipdf/v3/creator"
+	pdf "github.com/unidoc/unipdf/v3/model"
 )
 
 const (
-	RmX      = rm.Width
-	RmY      = rm.Height
-	A4X      = 210
-	A4Y      = 297
-	ratioA4X = float32(A4X) / float32(RmX)
-	ratioA4Y = float32(A4Y) / float32(RmY)
+	ratioA4X = float32(0.443)
```

Comment:
Not sure to understand how these ratio are calculated? Maybe an intermediate constant or a comment to explain it could make sense?

Reply:
This was based on trial and error. I'll revisit this to make sure we are not pulling it out of thin air :)